website/gatsby-config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
       resolve: `gatsby-plugin-react-svg`,
       options: {
         rule: {
-          include: /images\/.*\.svg/,
+          include: /images/,
```

Comment:
Why removing the more explicit filter?

Reply:
ah right, should i put it back in? I always have to remove it to get it working on my machine. I end up with exceptions like this: 
![image](https://user-images.githubusercontent.com/14233220/78500624-d0f81200-7757-11ea-9382-4c4252338413.png)